notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1114-SITCOMTN-092-Update_single_slew_analysis_to_use_total_Forces_and_Moments.ipynb

```diff
@@ -0,0 +1,2502 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    force_commands_to_plot = [
</pre><p>I suggest to pass this as an argument of analyze_m1m3_ics_slew_event.</p><p>Global variable used in function should be avoided</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Done</p>